src/Shared/Recruit.Vacancies.Client/Ioc/ServiceCollectionExtensions.cs

```diff
@@ -41,60 +41,73 @@ public static void AddRecruitStorageClient(this IServiceCollection services, ICo
         {
             services.Configure<AccountApiConfiguration>(configuration.GetSection("AccountApiConfiguration"));
 
-            RegisterAccountApiClientDeps(services);
-
             services.AddMediatR(typeof(CreateVacancyCommandHandler).Assembly);
             services.AddTransient<IMessaging, MediatrMessaging>();
             
+            // Vacancy Clients
             services.AddTransient<IEmployerVacancyClient, VacancyClient>();
```

Comment:
Wondering if there is a natural split between Employer and Jobs with regards the client? We have a QaVacancyClient. Is it possible to (and would we want to) have a JobsVacancyClient? One to discuss.